components/LessonContextCard.js

```diff
@@ -2,49 +2,66 @@ import styled from "styled-components";
 import { useState } from "react";
 
 const CardContainer = styled.div`
-  border: 1px solid black;
-  background-color: props.color;
-  width: 100%;
+  background-color: #fafafa;
+  margin: 0.8em;
+  padding: 0.6em;
+  border-radius: 10px;
   display: flex;
   flex-direction: column;
-  gap: 0.2em;
+  gap: 0.4em;
 `;
 
 const FirstTitle = styled.h3`
-  color: props.color;
+  color: #333;
 `;
 
-const Pattern = styled.div`
-  color: props.color;
+const Pattern = styled.div.attrs((props) => ({
+  $backgroundcolor: props.$backgroundcolor,
+}))`
+  color: #333;
+  background-color: ${(props) => props.$backgroundcolor};
+`;
```

Comment:
Given you're using an `onClick` attribute function on the calling of this element I'd recommend the usage of a `button` element to make it accessible for screen readers. 😊

Reply:
Thank you for that comment. I changed the code according to your suggestions :)